vscode-dotnet-runtime-library/src/test/unit/DotnetCoreAcquisitionWorker.test.ts

```diff
@@ -34,17 +36,19 @@ import {
 const assert = chai.assert;
 chai.use(chaiAsPromised);
 
+const expectedTimeoutTime = 6000;
+
 suite('DotnetCoreAcquisitionWorker Unit Tests', function () {
     const installingVersionsKey = 'installing';
     const installedVersionsKey = 'installed';
     const dotnetFolderName = `.dotnet O'Hare O'Donald`;
 
-    function getTestAcquisitionWorker(runtimeInstall: boolean, arch : string | null | undefined = undefined,
-        customEventStream? : MockEventStream , customContext? : MockExtensionContext): [DotnetCoreAcquisitionWorker, MockEventStream, MockExtensionContext]
+    function getTestAcquisitionWorker(runtimeInstall: boolean, arch? : string | null,

```

Comment:
I appreciate the change here. It makes thing simpler

Reply:
Thanks for your review! :)